modules/abstract-utxo/src/abstractUtxoCoin.ts

```diff
@@ -1442,7 +1442,11 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     };
   }
 
-  async getExtraPrebuildParams(buildParams: ExtraPrebuildParamsOptions): Promise<any> {
+  async getExtraPrebuildParams(buildParams: ExtraPrebuildParamsOptions & { wallet: Wallet }): Promise<any> {
```

Comment:
Is this new functionality covered in the transaction portion of the developer portal api docs?